sklearn/cluster/_optics.py

```diff
@@ -53,6 +53,9 @@ class OPTICS(ClusterMixin, BaseEstimator):
 
     Read more in the :ref:`User Guide <optics>`.
 
+    For an example of usage, see
+    :ref:`sphx_glr_auto_examples_plot_cluster_optics.py`.
```

Comment:
`WARNING: undefined label: 'sphx_glr_auto_examples_plot_cluster_optics.py'`

change it to:

```
    :ref:`sphx_glr_auto_examples_cluster_plot_optics.py`.
```

Reply:
@greyisbetter Thanks, I tried your suggestion but still there is 2 failing checks in the test.

Reply:
Using less extra spaces between paragraph may help. And I think it would be better if the example is at the end of the docstring.